terraform/service_rotation.tf

```diff
@@ -57,10 +57,17 @@ locals {
   ])
 }
 
+resource "google_secret_manager_secret_iam_member" "rotation-secrets-accessor" {
```

Comment:
I forgot that versionManager doesn't actually include permissions to access the secret. Oops.